### PR TITLE
Release CLI 2.7.0

### PR DESCRIPTION
**Pull request by bikmazefe**
Release CLI version 2.7.0 with GJT endpoints, following #281

### CODE REVIEW

File: clients/cli/go.sum [hunk 1]
```diff
@@ -220,6 +220,8 @@ github.com/pelletier/go-toml v1.8.1 h1:1Nf83orprkJyknT6h7zbuEGUEjcyVlCxSUGTENmNC
 github.com/pelletier/go-toml v1.8.1/go.mod h1:T2/BmBdy8dvIRq1a/8aqjN41wvWlN4lrapLU/GW4pbc=
 github.com/phrase/phrase-go/v2 v2.6.8 h1:PsEUUFE+6gaGPOXxl9L10iwMT/sR94YDGDrQTm41MVo=
 github.com/phrase/phrase-go/v2 v2.6.8/go.mod h1:N1ou4nnB9ak8aAWOH7x25Xhzw8FmFeBRhSGq5A0CY2o=
+github.com/phrase/phrase-go/v2 v2.7.0 h1:T3SgMkl+rciEVh29SvY5PCnubw1PTRf7Lhx3f8bzShs=
```
Comment (strzibny) [inline clients/cli/go.sum:223]: Why is this adding a new version instead of replacing it?

cc @theSoenke

Comment (jablan) [reply clients/cli/go.sum:223]: https://stackoverflow.com/questions/69825533/why-does-go-sum-include-so-many-older-packages

**Review by jablan — APPROVED** [review]

File: clients/cli/go.sum (hunk 1 above)
Comment (strzibny) [reply clients/cli/go.sum:223]: Thanks!

Comment (theSoenke) [reply clients/cli/go.sum:223]: `go mod tidy` will also cleanup unused versions. If you run `npm run generate.cli` it was removed for me as it is running the tidy step at the end

Comment (bikmazefe) [reply clients/cli/go.sum:221]: Updated it, thanks all! 🙌

**Review by strzibny — APPROVED** [review]

**Pull request merged**